cleanup/cleanDockerImages/CleanDockerImagesTest.groovy

```diff
@@ -0,0 +1,100 @@
+import groovy.json.JsonSlurper
+import org.apache.http.client.HttpResponseException
+import org.jfrog.artifactory.client.ArtifactoryClientBuilder
+import org.jfrog.artifactory.client.model.repository.settings.impl.DockerRepositorySettingsImpl
+import spock.lang.Specification
+
+class CleanDockerImagesTest extends Specification {
```

Comment:
Please add a unit test for the new feature

Reply:
unit tests are not working :D